### ISSUE TITLE
Date Picker Shows Sunday as First Day of Week

### ISSUE THREAD

**Issue by gohrner** [nextcloud/polls#567]
### What is going wrong?
**Describe the bug**

My user account uses German language settings which also causes the start-of-week day correctly to be set to Monday (instead of the unusual selection "Sunday", see http://chartsbin.com/view/41671) for the Nextcloud Calendar app.

In "Polls", the date picker always shows Sunday as the first day of the week, which is horribly confusing and leads to frequent errors.

**To Reproduce**
1. Switch NextCloud to German language (or probably most locales besides American ones)
1. Create a new poll
1. Try to add a date

**Expected behavior**

The date picker should pop up with Monday being the first day of the week.

**Actual Behaviour**

The date picker pops up with Sunday being the first day of the week.

### Information about your polls installation
**Polls version?** (see apps page)

0.10.2

**Fresh installation or update from a prior version (from which one)?**

0.9.x IIRC - via 0.10.x

**How did you install this version?(Appstore or describe installation)**

Appstore

### Information about your Instance of Nextcloud/ownCloud
**Nextcloud or ownCloud?**

NextCloud

**Which Version?**

15.0.7

**Nextcloud/ownCloud version:** (see Nextcloud admin page)

???

**List of activated apps:**

```
Enabled:
  - accessibility: 1.1.0
  - activity: 2.8.2
  - calendar: 1.6.4
  - cloud_federation_api: 0.1.0
  - comments: 1.5.0
  - contacts: 3.1.0
  - dav: 1.8.1
  - federatedfilesharing: 1.5.0
  - federation: 1.5.0
  - files: 1.10.0
  - files_pdfviewer: 1.4.0
  - files_sharing: 1.7.0
  - files_texteditor: 2.7.0
  - files_trashbin: 1.5.0
  - files_versions: 1.8.0
  - files_videoplayer: 1.4.0
  - firstrunwizard: 2.4.0
  - gallery: 18.2.0
  - impersonate: 1.2.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.3.0
  - mail: 0.11.1
  - nextcloud_announcements: 1.4.0
  - notifications: 2.3.0
  - oauth2: 1.3.0
  - password_policy: 1.5.0
  - polls: 0.10.2
  - provisioning_api: 1.5.0
  - richdocuments: 3.2.4
  - serverinfo: 1.5.0
  - sharebymail: 1.5.0
  - support: 1.0.0
  - survey_client: 1.3.0
  - systemtags: 1.5.0
  - theming: 1.6.0
  - twofactor_backupcodes: 1.4.1
  - updatenotification: 1.5.0
  - workflowengine: 1.5.0
Disabled:
  - admin_audit
  - encryption
  - files_external
  - user_ldap
```

### Server configuration

**Operating system**: Debian 9 Stretch

**Web server:** Apache 2.4.25-3+deb9u7

**Database:** MariaDB 10.1.37-0+deb9u1

**PHP version:** 7.0.33-0+deb9u3

**Signing status:**

```
No errors have been found.
```

**Are you using an external user-backend, if yes which one:**
none

### Client configuration
**Device:**
Desktop

**Browser:**

Chromium
Version 73.0.3683.86 (Offizieller Build)

**Operating system:**

Ubuntu 18.04

**Comment by dartcafe** [comment]
Still an issue of the vue datepicker. @skjnldsv

**Comment by skjnldsv** [comment]
@dartcafe you need to specify the first day to the datepicker. 
https://github.com/nextcloud/nextcloud-vue/pull/146/files

**Comment by dartcafe** [comment]
I was hoping https://github.com/nextcloud/nextcloud-vue/pull/146 could manage this, because it depends on the locale setting of the user.

**Comment by skjnldsv** [comment]
@dartcafe it could, that is indeed the purpose of this pr, but we're not really happy about the server side implementation, so in the meantime, you need to implement it from your app ;)

**Comment by dartcafe** [comment]
Hmmm grmpfl....

**Comment by skjnldsv** [comment]
@dartcafe :grin: 
Just copy/paste the content from the component's pr :)

**Comment by dartcafe** [comment]
Fixed for 1.0
